cfgov/cfgov/settings/base.py

```diff
@@ -264,6 +265,8 @@
     PROJECT_ROOT.child('templates', 'wagtailadmin')
 ]
 
+static_in = REPOSITORY_ROOT.child('static.in')
+STATICFILES_DIRS += [str(d) for d in static_in.listdir(filter=DIRS)]
```

Comment:
If you add this logic into the base settings, can you also remove [this duplicative line](https://github.com/cfpb/cfgov-refresh/blob/49aa3e23981e983db99f1788d24e542a390a72d1/cfgov/cfgov/settings/local.py#L16) from local settings?